artscraper/artist_works_metadata.py

```diff
@@ -95,3 +97,133 @@ def get_artist_description(artist_link):
     description = page.summary
 
     return description
+
+def get_artist_metadata(artist_link):
+
+    '''
+    Parameters
+    ----------
+    artist_link : Web address of the artist's Google Arts & Culture page
+
+    Returns
+    -------
+    metadata: Dictionary containing metadata about the artist
+    '''
+
+    # Get Wikidata ID of artist
+    artist_id = get_artist_wikidata_id(artist_link)
+
+    # Wikidata database to query
+    url = 'https://query.wikidata.org/sparql'
+
+    # SPARQL query
+    with open('../docs/sparql_query.txt', 'r', encoding='utf-8') as file:
+        query = file.readlines()
+
+    # Replace person_id in SPARQL query with the wikidata ID of the artist
+    query = [q.replace('person_id', artist_id) for q in query]
+
+    # Send query request
+    request = requests.get(url, params= {'format': 'json', 'query': ''.join(query)}, timeout=10)
+
+    # Convert response to dictionary
+    data = request.json()
+
+    # Assemble metadata in a dictionary
+    metadata = {
+        'family name': get_single_valued_property(data, 'familyName'),
+        'given name': get_single_valued_property(data, 'givenName'),
+        'date of birth': get_single_valued_property(data, 'dateOfBirth').rsplit('T')[0],
+        'place of birth': get_single_valued_property(data, 'placeOfBirth'),
+        'latitude of place of birth' :
+            round(float(get_single_valued_property(data, 'latitudeBirth')),2),
```

Comment:
Why is it rounded here? I would suggest to round it later if you want to display it, but keep the non-rounded numbers everywhere else.

Reply:
I have now removed the rounding.